server/config.py

```diff
@@ -1,11 +1,19 @@
 import os
+import datetime
 
+# Env variables
 INNER_PORT = os.getenv("INNER_PORT")
 POSTGRES_DB_NAME = os.getenv("POSTGRES_DB_NAME")
 POSTGRES_USERNAME = os.getenv("POSTGRES_USERNAME")
 POSTGRES_HOST = os.getenv("POSTGRES_HOST")
 POSTGRES_PORT = os.getenv("POSTGRES_PORT")
 POSTGRES_PASSWORD = os.getenv("POSTGRES_PASSWORD")
+DEBUG = os.getenv("DEBUG") != "false"
+
+CONFERENCE_EXPIRATION_TIME = (
+    datetime.timedelta(hours=2) if not DEBUG else datetime.timedelta(minutes=5)
```

Comment:
I think it must be documented somewhere

Reply:
Where?

Reply:
You might make `CONFERENCE_EXPIRATION_TIME` an environment variable since I don't think it should be hard-coded like this. Furthermore, if you want to set the parameter in the DEBUG mode, say, for testing in Docker, it's better to create both `CONFERENCE_EXPIRATION_TIME` and `DEBUG_CONFERENCE_EXPIRATION_TIME` environment variables

Reply:
Also, if only you used the pydantic `AppSettings` class, you might have explicitly informed the users if no such environment variable provided. So it'd be already kind of documentation, wouldn't it?

Reply:
Fixed